srcpkgs/libcaca/template

```diff
@@ -11,17 +11,16 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="WTFPL"
 homepage="http://caca.zoy.org/wiki/libcaca"
 distfiles="https://github.com/cacalabs/libcaca/archive/v${version}.tar.gz"
-checksum=7ed29a00cc7f017424d8b2994f001f137ed5bc4441987b711d78c6734fdf3493
+checksum="3edb8763a8f888ed4d4b85b3a056e81c840d5d27f79bdebc0b991688b23084f2"
 
 build_options="x11 opengl"
 
 pre_configure() {
-	vsed -i -e 's,AM_CONFIG_HEADER,AC_CONFIG_HEADERS,' configure.ac
 	autoreconf -fi
 }
 
 post_install() {
-	rm ${DESTDIR}/usr/lib/*.la
+	rm -f ${DESTDIR}/usr/lib/*.la
```

Comment:
don't use -f, it obscures useless removals

Reply:
OK, will correct that.